mobify/progressive-web-scaffold
# Remove the `build` directory on `prod:build`

The `build` directly doesn't get cleared out when you do a production build - this has been a sneaky problem for me in the past. 

## Changes
- When running `prod:build`, `build/` is removed

## How to test-drive this PR
- Without this PR, run `prod:build`, put some random file(s) in `build/`, run `prod:build` again, observe those files are still around
- With this PR, perform the above again, observe the random files are gone in the end
